zxinglight/__init__.py

```diff
@@ -67,7 +67,7 @@ class BarcodeType(IntEnum):
     UPC_EAN_EXTENSION = 17
 
 
-def read_codes(image, barcode_type=BarcodeType.NONE, try_harder=False, hybrid=False):
+def read_codes(image, barcode_type=BarcodeType.NONE, try_harder=False, hybrid=False, multi=False):
```

Comment:
What about setting `multi` to `True` by default? I'd prefer to preserve backwards compatibility.

Reply:
Done and done.